app/models/address_model.py

```diff
@@ -0,0 +1,16 @@
+from app.configs.database import db
+from sqlalchemy import Column, ForeignKey, Integer, String
+from dataclasses import dataclass
+
+
+@dataclass
+class AddressModel(db.Model):
+    id: int
```

Comment:
Do we need to show id when requesting the address? I don't think this is necessary since the we will probably use the name to make this relationship.

Reply:
Done!